projects/packages/jetpack-mu-wpcom/src/features/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-launchpad-navigator.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Launchpad API endpoint
+ *
+ * @package automattic/jetpack-mu-wpcom
+ * @since 1.1.0
```

Comment:
I think this should be `4.9.0`, as our last release was `4.8.0`.
```suggestion
 * @since 4.9.0
```